### DISCUSSION TITLE
Release Strategy

### DISCUSSION THREAD

**Discussion by mgilangjanuar** [mgilangjanuar/teledrive#17] [Announcements]
Hello everyone!

I have a plan to release this project to the public. I'll create the coming soon page with an email subscription form on it. So, we can release before the product has done 🤣

![gif](https://i.giphy.com/media/Ssltx68WIeX1wA0Mg8/giphy-downsized.gif)

Any idea (or template) for this page? 🤣🤣🤣

**Comment by Aqshola** [comment]
[Chakra template dev](https://chakra-templates.dev/page-sections/hero)  come to rescue (again)🤣, there's several  hero component then add  custom subs form

**Reply by Aqshola** [reply]
I found another [BWA lentera](https://pixel.buildwithangga.com/details/lentera-header-website) but we will need translate to chakra ui

**Reply by mgilangjanuar** [reply]
Whoah cool, thank you

**Comment by mgilangjanuar** [comment]
<img width="1348" alt="image" src="https://user-images.githubusercontent.com/3146378/132041985-4d3f35f5-5394-4db6-90a9-3eae273d633c.png">

It's here: [teledrive.vercel.app](https://teledrive.vercel.app) 🎊
